apps/emqx/src/emqx_schema.erl

```diff
@@ -1777,7 +1777,7 @@ fields("session_persistence") ->
             sc(
                 pos_integer(),
                 #{
-                    default => 1000,
+                    default => 100,
```

Comment:
Since the introduction of the mqueue buffer for persistent sessions, it's no longer necessary to use huge batches.